doc/guides/writing-tests.md

```diff
@@ -209,6 +209,35 @@ const assert = require('assert');
 const freelist = require('internal/freelist');
 ```
 
+### Assertions
+
+When writing an assertion for comparison, prefer the strict versions:
```

Comment:
Hmmm, yes, the word "for" does sound wrong here, or the order of `assertion` and `comparison` here is wrong. Probably `assertion with comparison`?